src/Classes/ItemsTab.lua

```diff
@@ -2563,8 +2563,8 @@ function ItemsTabClass:AddItemTooltip(tooltip, item, slot, dbMode)
 
 	-- Requirements
 	self.build:AddRequirementsToTooltip(tooltip, item.requirements.level, 
-		item.requirements.strMod, item.requirements.dexMod, item.requirements.intMod, 
-		item.requirements.str or 0, item.requirements.dex or 0, item.requirements.int or 0)
+		item.requirements.strMod, item.requirements.dexMod, item.requirements.intMod, item.requirements.omniMod, 
+		item.requirements.str or 0, item.requirements.dex or 0, item.requirements.int or 0, item.requirements.omni or 0)
```

Comment:
I don't think this is supposed to be here, or you forgot to add these params to the method signature?

Reply:
Addressed this to properly do the tooltip requirement for Omniscience 
![image](https://user-images.githubusercontent.com/17045310/152023729-4ec56365-7a3b-40e1-8396-c7ca9ffd567d.png)